image_explorer/image_explorer.py

```diff
@@ -201,6 +209,7 @@ def studio_view(self, context):
     def studio_submit(self, submissions, suffix=''):
 
         self.display_name = submissions['display_name']
+        self.hot_spots = ast.literal_eval(submissions['hot_spots'])
```

Comment:
We should not be using eval. Since `submissions` is parsed from submitted JSON, we can just make sure that the JSON includes a boolean rather than a string.